nifi-commons/nifi-property-protection-gcp/pom.xml

```diff
@@ -53,6 +53,7 @@
         <dependency>
             <groupId>com.google.cloud</groupId>
             <artifactId>google-cloud-kms</artifactId>
+            <version>2.4.0</version> 
```

Comment:
This version change should be removed. The `google-cloud-kms` version is managed through the `libraries-bom` dependency.